Cargo.toml

```diff
@@ -33,3 +33,4 @@ rand_core = { git = "https://github.com/servo/rand", branch = "servo-rand_os-0.1
 autocfg = { git = "https://github.com/servo/autocfg", branch = "rustflags" }
 # https://github.com/retep998/winapi-rs/pull/816
 winapi = { git = "https://github.com/servo/winapi-rs", branch = "patch-1" }
+sparkle = { git = "https://github.com/servo/sparkle", branch = "master" }
```

Comment:
Why?

Reply:
There have been a few new commits since the last published version of sparkle (ie. servo/sparkle#8 has landed). I can remove it when @jdm increases the version.

Reply:
Published 0.1.7.